molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -39,7 +39,7 @@
         path: "{{ molecule_ephemeral_directory }}"
         name: "molecule_local/{{ item.item.image }}"
         docker_host: "{{ item.item.docker_host | default(lookup('env', 'DOCKER_HOST') or 'unix://var/run/docker.sock') }}"
-        dockerfile: "{{ item.item.dockerfile | default(item.invocation.module_args.dest) }}"
```

Comment:
.dockerfile is the template not the generated file, this was the mistake. this is mainlt reverting it to old value.